### PR TITLE
Updates for September 2023

### LINKED ISSUE #59: Update about page
### Describe the solution

I haven't touched the about page since the inception. We need updated information to it. I wonder if he still offer thos stickers...

This also includes our team members, which can be moved to the org page if it is done.

### LINKED ISSUE #60: Copy contents of the comeback post on Reddit to about page
### Describe the solution

Something tells me that we can copy the contents of the post sent by @Codixer on Reddit to the about page...

https://new.reddit.com/r/placeAtlas2023/comments/154z3q4/reddit_is_restarting_rplace_for_2023_were_back_as/

### LINKED ISSUE #247: Add The Final Clean canvas
### Describe the solution

Just like last year, we should include The Final Clean canvas to the page.

Checklist:

- [x] Wait until TFC is released
- [x] Compress the images
- [x] Push to GitHub

### LINKED ISSUE #290: Add support of TemplateManager
### Describe the solution

Not sure if it is worth the addition, but I'm thinking to add support of templates for TemplateManager, so people can draw to it based on the template.

Again, it is still needs to be discussed whether if it is worth adding it (TFC would have it cleaned, anyway). I think this would be easy to add, though.

- https://github.com/osuplace/templateManager
- JSON example: https://pride.place/template.json
- How to: https://docs.google.com/document/d/12WTiDcRo4P35zJvlgWX06MKVbitbDo3ehnF7mysFv4Y/edit?rm=minimal
- Generator: https://charity.pxls.space/

### PR DESCRIPTION
**Pull request by Hans5958**
Few code changes for updates to the Atlas. The primary focus is the inclusion of TFC, but other changes can be vetted.

Resolves #59
Resolves #60
Resolves #247
Resolves #290

### CODE REVIEW

File: tools/extend_entry_times.py [hunk 1]
```diff
@@ -3,22 +3,22 @@
 import os
 import tqdm
 
-pre_extend_times = [250, 254, 255, 256, 257]
-post_extend_time = 258
+# pre_extend_times = [250, 254, 255, 256, 257]
```
Comment (AnonymousRandomPerson) [inline tools/extend_entry_times.py:6]: I use this script every time I update the atlas, so all the commented-out lines should be added back.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
It's not possible to proceed after adding a TFC period and drawing a polygon.

**Comment by Hans5958** [comment]
> It's not possible to proceed after adding a TFC period and drawing a polygon.

Try again now?

File: tools/extend_entry_times.py [hunk 2]
```diff
@@ -40,29 +40,43 @@ def per_line_entries(entries: list, file: TextIOWrapper):
...
+				variation = time[:time.find(':')]
+				time = time[time.find(':') + 1:]
 			if '-' in time:
-				end_time = time[time.find('-') + 1:]
+				start_time = int(time[:time.find('-')]) 
+				end_time = int(time[time.find('-') + 1:])
 			else:
-				end_time = time
-			if int(end_time) in pre_extend_times:
+				start_time = end_time = int(time)
```
Comment (AnonymousRandomPerson) [inline tools/extend_entry_times.py:64]: This breaks on the updated atlas.json with the TFC `T` times.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]

File: tools/extend_entry_times.py [hunk 3]
```diff
@@ -40,29 +40,50 @@ def per_line_entries(entries: list, file: TextIOWrapper):
 
 def extend_time_key(items):
 	for key, value in list(items.items()):
+		old_key = key
```
Comment (AnonymousRandomPerson) [inline tools/extend_entry_times.py:43]: Running this script on the current atlas.json in the PR appends an extra `, ` to all times. There should be no changes if the atlas.json is already extended.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]

File: tools/extend_entry_times.py (hunk 3 above)
Comment (Hans5958) [reply tools/extend_entry_times.py:43]: Turns out my solution may have more problems than as it seems. Have to fix it further.

**Review by AnonymousRandomPerson — APPROVED** [review]

**Pull request merged**